### PR TITLE
Refactor setting of Simulator run options

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
Did refactoring of Simulator run options as per @jlipps's comment for https://github.com/appium/appium-ios-simulator/pull/112

Please do not merge this PR until the other PR which contains the necessary changes in xcuitest driver is done.

### CODE REVIEW

File: lib/simulator-xcode-6.js [hunk 1]
```diff
@@ -291,21 +264,32 @@ class SimulatorXcode6 extends EventEmitter {
     return indicator;
   }
 
-  async run (startupTimeout = this.startupTimeout, allowTouchEnroll = false) {
+  async run (startupTimeout = this.startupTimeout, opts = {}) {
```
Comment (jlipps) [inline lib/simulator-xcode-6.js]: why not make `startupTimeout` an opt as well?

**Comment by mykola-mokhnach** [comment]
@jlipps I've also moved the timeout arguments under opts. Please take another look into this

**Comment by imurchie** [comment]
One small formatting issue and then this should be good to go.
```
/Users/travis/build/appium/appium-ios-simulator/lib/simulator-xcode-6.js
  66:26  error  Missing space before function parentheses  space-before-function-paren
```

**Comment by mykola-mokhnach** [comment]
@imurchie thanks, fixed

**Comment by imurchie** [comment]
Published as `appium-ios-simulator@2.0.0`!

**Pull request merged**